src/documents/ComponentConfigurationOptions.md

```diff
@@ -468,19 +446,51 @@ Component developers are free to define their own additional options.
   </tr>
 </table>
 
+### `styles` ###
+<table>
+  <tr>
+    <th>Description</th>
+    <td>A record containing named CSS classes that the component will apply to its markup in order to achieve state-dependent styling effects.</td>
+  </tr>
+  <tr>
+    <th>Notes</th>
+    <td>The contents of this block are not interpreted by the framework at all. The existence of this block amounts to a helpful convention that implementors of view components are recommended to use, to organise
+    and advertise the CSS class names that they will apply on behalf of their users</td>
+  </tr>
+  <tr>
+    <th>Example Definition</th>
+    <td><pre>
+fluid.defaults("demo.initGridReorderer", {
+    gradeNames: ["fluid.reorderGrid"],
+    styles: {
+        dragging: "demo-gridReorderer-dragging",
+        avatar: "demo-gridReorderer-avatar",
+        selected: "demo-gridReorderer-selected",
+        dropMarker: "demo-gridReorderer-dropMarker"
+    },
+    disableWrap: true
+});
+</pre></td>
+</tr>
+
+</table>
+
+In addition to the options above, a View Component also accepts an additional argument named `container` which may be supplied either as the first argument to its [Creator Function](UnderstandingInfusionComponents.md)
+or else at top level in its [Subcomponent Record](SubcomponentDeclaration.md). It is not currently supported to supply this value as a standard option in the options record.
 
 ## Renderer Components ##
 
 Components defined with a grade of `rendererComponent` are also view components (and hence model components and evented components), so they support
 * all of the [common options](#options-supported-by-all-components-grades) described above,
 * [`modelComponent` options](#model-components) described above,
-* [`eventedComponent` options](#evented-components) described above,
 * [`viewComponent` options](#view-components) described above,
 * and those defined below.
 
 Component developers are free to define their own additional options.
 
-### selectorsToIgnore ###
+**NOTE**: The Infusion Renderer system will be rewritten completely for the Infusion 2.0 release - the use of the current renderer and component hierarchy is not recommended.
```

Comment:
Again, can we be more specific, at least by saying "rewritten completely before the final release of Infusion 2.0?" It's this kind of issue that makes me increasingly wonder if we should indeed be more frequently updating our major revision numbers. The question of "when" is very fuzzy with our current plan to issue a long series of not-fully-roadmapped betas.